### ISSUE TITLE
Can this be used for any service

### ISSUE THREAD

**Issue by amulyas** [cf-platform-eng/cloudformation-broker#1]
Hey @frodenas can this be used for any service for example let I have cloud formation to create ELK cluster can I use this broker to create ELK as a service .. ?

**Comment by frodenas** [comment]
@amulyas Yes, it can be any service supported by cloudformation. The `credentials` hash returned by the broker will be the `stack` output values.

**Comment by amulyas** [comment]
Thanks @frodenas your code rocks
